site/gatsby-site/src/components/RelatedIncidentsArea.js

```diff
@@ -80,19 +82,21 @@ const RelatedIncidentsArea = ({
               <span>
                 {val?.incident_id && (
                   <span className="text-lg">
-                    <LocalizedLink
-                      to={'/cite/' + val.incident_id}
+                    <a
```

Comment:
I'm guessing `LocalizedLink` doesn't support the `target` prop?

Reply:
I don't believe so – it didn't work until I changed it to an `<a>`.

Reply:
Seems they fixed it on Gatbsy v5 but we aren't there yet.